cmd/bosun/main.go

```diff
@@ -282,6 +282,10 @@ func main() {
 	}()
 
 	if *flagWatch {
+		// note for vim users, I've this in my .vimrc:
```

Comment:
Don't think this should be a comment in main.go. Maybe add a tips section to https://github.com/bosun-monitor/bosun/blob/master/CONTRIBUTING.md